_regional/pybeach.yml

```
@@ -0,0 +1,5 @@
+---
+name: PyBeach
+location: Los Angeles, California, United States
+website: https://pybeachorg/
```

```suggestion
website: https://pybeach.org/
```
Just a small typo, and this one is good to go!

---

🤦‍♂ Thanks for pointing that out! It's only the most important part of the PR… 😬